docs/Contributing/Seeding-Data.md

```diff
@@ -2,47 +2,80 @@
+export TOKEN=eyJhbGciOi... # your api token
 ```
 
-Next, set the `FLEET_ENV_PATH` to point to the `env` file. This will let the scripts in the `fleet/` folder source the env file.
+Conveniently, you can store the `env` file as `fleet_env` in the `./code/fleet-api-env` directory.
+
+## Apply your environment variable file
+
+Next, set the `FLEET_ENV_PATH` to point to the `env` file using the command line.
 
 ```
-export FLEET_ENV_PATH=/Users/victor/fleet_env
+export FLEET_ENV_PATH=./code/fleet-api-env/fleet_env
 ```
 
-Finally, run one of the bash scripts located in the [/tools/api](../../tools/api/README.md) directory.
+This will let the scripts in the `/tools/api/fleet/` folder source the `env` file.
+
+## Run one of the bash scripts to seed the data
+
+Finally, run one of the bash scripts located in the [/tools/api](../../tools/api) directory.
+
+### Seed free users
 
 The `fleet/create_free` script will generate an environment to roughly reflect an installation of Fleet Free. The script creates three users with different roles.
 
 ```
 ./tools/api/fleet/teams/create_free
 ```
 
-The `fleet/create_premium` script will generate an environment to roughly reflect an installation of Fleet Premium. The script will create two teams of four users with different roles.
+<img width="1000" alt="Test users for a free account" src="https://user-images.githubusercontent.com/71795832/191982367-f89b3d6e-085b-4be3-be64-f3e725d31fa5.png">
+
+### Seed premium users
+
+The `fleet/create_premium` script will generate an environment to roughly reflect an installation of Fleet Premium. The script will create two teams of seven users with different roles.
 
 ```
 ./tools/api/fleet/teams/create_premium
 ```
 
+<img width="1000" alt="Test users for a premium account" src="https://user-images.githubusercontent.com/71795832/191981351-930d7f4d-5343-4fc3-974a-07238456ed26.png">
+
+### Seed designing users
+
 The `fleet/create_figma` script will generate an environment to reflect the mockups in the Fleet EE (current) Figma file. The script creates three teams and twelve users with different roles.
 
 ```
 ./tools/api/fleet/teams/create_figma
 ```
 
-Each user-generated by the script has its password set to `user123#`.
+<img width="1000" alt="Users for designing in Figma" src="https://user-images.githubusercontent.com/71795832/191983900-1df77482-1c2c-47f0-a36f-bf20fcc68a61.png">
 
-In order to run scripts that make use of premium features, make sure you have started the server with the correct flags as described in [Testing](./Testing-and-local-development.md#license-key).
+Each user-generated by the script has its password set to `password123#`.
 
 <meta name="pageOrderInSection" value="600">
+
+## Related actions
+
+### Import queries
+
+After seeding users and teams, checkout [importing queries](../using-fleet/standard-query-library#importing-the-queries-in-fleet) from Fleet's [standard query library](https://fleetdm.com/queries).
```

Comment:
Not sure this relative link will work.

Reply:
Can you elaborate why? I see 123 instances of relative links in the `/docs/` folder so I assume this one will work (probably so it's usable in VS code, github, and on the marketing site).

@chris-mcgillicuddy could you shed some light on relative links?

Reply:
I think the relative URL needs to include `/docs` like this `/docs/using-fleet/standard-query-library#importing-the-queries-in-fleet`. To be safe, we could include the full URL `https://fleetdm.com/docs/using-fleet/standard-query-library#importing-the-queries-in-fleet`.

Reply:
Or, if you are going to use relative links, then it should be `../../`. I see a lot of folks are still using relative links in recent commits. They do work, so long as the page is never moved. IMO it's safer to always use absolute to avoid future problems.

Reply:
@lukeheath there should be a ticket to fix all relative links to absolute links so this is not an ongoing discussion, especially if we're asking every oncall engineer to update docs and other's will follow patterns they see like I did.